<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../sprites/icon.ico</normaloff>../sprites/icon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="aboutTab">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;aarctic 1.1 &lt;/span&gt;- Simplistic desktop client of &lt;a href=&quot;http://aarddict.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#009dff;&quot;&gt;Aard2&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Made with &amp;lt;3 by &lt;a href=&quot;https://github.com/aisuneko&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#009dff;&quot;&gt;aisuneko&lt;/span&gt;&lt;/a&gt;, licensed under GPLv3&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;aarctic&lt;/span&gt; is brought to you by PyQt 5.15.7 / Qt 5.15.5, &lt;br/&gt;plus the following libraries:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Slob&lt;/span&gt; - &lt;a href=&quot;https://github.com/itkach/slob&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#009dff;&quot;&gt;github.com/itkach/slob&lt;/span&gt;&lt;/a&gt; (GPLv3)&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Bottle &lt;/span&gt;- &lt;a href=&quot;https://bottlepy.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#009dff;&quot;&gt;bottlepy.org&lt;/span&gt;&lt;/a&gt; (MIT)&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;requests &lt;/span&gt;- &lt;a href=&quot;https://requests.readthedocs.io&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#009dff;&quot;&gt;requests.readthedocs.io&lt;/span&gt;&lt;/a&gt; (Apache 2.0)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dictInfoTab">
      <attribute name="title">
       <string>Installed Dictionaries</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="dictInfoView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
